cypress/Shared/MeasuresPage.ts

```diff
@@ -27,26 +37,62 @@ export class MeasuresPage {
             filePath = 'cypress/fixtures/measureId2'
         }
 
+        //block of code that will be used once the flag is removed
+        /*         cy.readFile(filePath).should('exist').then((fileContents) => {
+                    Utilities.waitForElementVisible('[data-testid=measure-action-' + fileContents + ']', 30000)
+                    cy.get('[data-testid=measure-action-' + fileContents + ']').should('be.visible')
+                    Utilities.waitForElementEnabled('[data-testid=measure-action-' + fileContents + ']', 30000)
+                    cy.get('[data-testid=measure-action-' + fileContents + ']').should('be.enabled')
+                    cy.get('[data-testid=measure-action-' + fileContents + ']').click()
+                    Utilities.waitForElementVisible('[data-testid=view-measure-' + fileContents + ']', 30000)
+                    cy.get('[data-testid=view-measure-' + fileContents + ']').should('be.visible')
+                    Utilities.waitForElementEnabled('[data-testid=view-measure-' + fileContents + ']', 30000)
+                    cy.get('[data-testid=view-measure-' + fileContents + ']').should('be.enabled')
+                    cy.get('[data-testid=view-measure-' + fileContents + ']').click()
+        
+                }) */
         cy.readFile(filePath).should('exist').then((fileContents) => {
             Utilities.waitForElementVisible('[data-testid=edit-measure-' + fileContents + ']', 30000)
             cy.get('[data-testid=edit-measure-' + fileContents + ']').should('be.visible')
             Utilities.waitForElementEnabled('[data-testid=edit-measure-' + fileContents + ']', 30000)
             cy.get('[data-testid=edit-measure-' + fileContents + ']').should('be.enabled')
             cy.get('[data-testid=edit-measure-' + fileContents + ']').click()
+        })
+    }
+    public static clickEditforCreatedDraft(secondDraft?: boolean): void {
```

Comment:
I don't think that we need this, but maybe there is a reason I am not thinking of?

Reply:
Removed this function